manager/integration/tests/test_ha.py

```diff
@@ -1919,3 +1919,22 @@ def test_rebuild_after_replica_file_crash(client, volume_name): # NOQA
     assert crashed_replica["failedAt"] is not None
 
     check_volume_data(volume, data)
+
+
+@pytest.mark.skip(reason="TODO")
+def test_extra_replica_cleanup():
+    """
+    Test extra failed to scheduled replica cleanup
+    when no eviction requested
+
```

Comment:
Mention `Disable soft anti-affinity` explicitly

Reply:
Sure, will add it at the beginning.